layouts/partials/comments/provider/gitalk.html

```diff
@@ -0,0 +1,25 @@
+{{- with .Site.Params.comments.gitalk -}}
+
+<div id="gitalk-container"></div>
+<link rel="stylesheet" href="https://unpkg.com/gitalk/dist/gitalk.css">
+<script src="https://unpkg.com/gitalk/dist/gitalk.min.js"></script>
+<script src="https://cdn.bootcdn.net/ajax/libs/blueimp-md5/2.18.0/js/md5.js"></script>
```

Comment:
Since the most of third-party resources of this theme are fetched from JSDelivr, I think it's better to switch to JSDelivr.